commands/bootstrap.js

```diff
@@ -55,13 +60,58 @@ function api({
 	allsamples = false,
 	link = true,
 	loglevel = 'error',
+	override,
 	verbose = false
 } = {}) {
 	const pkg = packageRoot(cwd);
 
 	if (verbose) loglevel = 'verbose';
 
 	return Promise.resolve()
+		.then(() => {
+			// Override dependencies in package.json, package-lock.json and npm-shrinkwrap.json
+			if (override) {
+				console.log('Overrides with local dependencies from', override);
+				// Collect list of all valid local package tgz archives found
+				const scoped = s => fs.readdirSync(path.join(override, s)).map(d => path.join(s, d));
+				const local = fs
+					.readdirSync(override)
+					.reduce((a, curr) => a.concat(curr.startsWith('@') ? scoped(curr) : curr), [])
+					.filter(d => fs.existsSync(path.join(override, d, 'package.tgz')));
+				if (path.isAbsolute(override)) override = path.relative(cwd, override);
+				['package.json', 'package-lock.json', 'npm-shrinkwrap.json']
+					.map(f => path.join(cwd, f))
+					.filter(f => fs.existsSync(f))
+					.forEach((f, i) => {
+						const lockfile = i > 0;
+						// Restore any detected backups
+						if (fs.existsSync(f + '.bak')) {
+							fs.unlinkSync(f);
+							fs.renameSync(f + '.bak', f);
+						}
+						const obj = JSON.parse(fs.readFileSync(f, {encoding: 'utf8'}));
+						// Update dependency entry for local entries that exist
+						local
+							.filter(dep => obj.dependencies && obj.dependencies[dep])
+							.forEach(dep => {
+								if (lockfile) {
+									obj.lockfileVersion = obj.lockfileVersion || 1;
+									obj.requires = true;
+									obj.dependencies[dep].version = `file:${path.join(override, dep, 'package.tgz')}`;
+									// Remove unneeded properties to avoid issues
+									['resolved', 'from', 'integrity', 'requires'].forEach(
+										key => delete obj.dependencies[dep][key]
+									);
+								} else {
+									obj.dependencies[dep] = `file:${path.join(override, dep, 'package.tgz')}`;
```

Comment:
Really minor nit, could we extract the path to `package.tgz` outside the `if`?

Reply:
Can do 👍